interface/forms/misc_billing_options/new.php

```diff
@@ -69,7 +67,7 @@
     </script>
   </head>
   <body class="body_top">
-    <form method='post' <?php echo "name='my_form' " . "action='$rootdir/forms/misc_billing_options/save.php?id=" . attr($formid) . "'>\n";?>>
```

Comment:
missing the ending tag in the form html here. Note it's a bit confusing since the `?>` is for ending the php element. But still need a `>` to end the form html element.